packages/wrangler/src/utils/stdout-styling.ts

```diff
@@ -0,0 +1,17 @@
+import chalk from "chalk";
+import supportsColor from "supports-color";
+
+/**
+ * Terminal styling util function that sets text color. If terminal does
+ * not support colors, it will return the unchanged text instead.
+ */
+export function highlight(text: string, colorHex: string): string {
+	return supportsColor.stdout ? chalk.hex(colorHex)(text) : text;
+}
+
+/**
+ * Terminal styling util function that makes text bold.
+ */
+export function embolden(text: string): string {
```

Comment:
happy for other name suggestions

Reply:
What about just importing `bold` from `chalk` and using that directly? Is this wrapper needed?

Reply:
yeah, since we don't need `highlight` anymore either, I'll remove this too